packaging/arch/PKGBUILD

```diff
@@ -14,19 +15,25 @@ checkdepends=('python' 'squashfs-tools' 'indent' 'shellcheck')
 
 options=('!strip' 'emptydirs')
 install=snapd.install
-source=("git+https://github.com/snapcore/$pkgname.git")
+source=("git+https://github.com/bboozzoo/$pkgbase.git")
```

Comment:
This should not be changed.

Reply:
Thanks for spotting this

Reply:
If nobody minds, I'll fixup and force push an update for this.